docs/resources/rest_managed.md

```diff
@@ -94,7 +97,7 @@ resource "aci_rest_managed" "example_tenant_with_child" {
 
   #### Optional ###
 
-  * `content` (map) Map of key-value pairs which represents the attributes for the child object. When annotation is provided in the content of the child it will take precedence over the annotation set at the parent level.
+  * `content` - (map) Map of key-value pairs which represents the attributes for the child object. When annotation is provided in the content of the child it will take precedence over the annotation set at the parent level.
```

Comment:
-> A map of key-value pairs represents the attributes of the child object. Annotation defined within the child's content take precedence over annotation set at the parent level.

Reply:
fixed